package/src/bin/omicron-package.rs

```diff
@@ -937,6 +972,67 @@ impl Config {
             _ => bail!("Aborting"),
         }
     }
+
+    /// Returns target packages to be assembled on the builder machine, limited
+    /// to those specified in `only` (if set).
+    fn packages_to_build(&self) -> PackageMap<'_> {
+        let packages = self.package_config.packages_to_build(&self.target);
+        if self.only.is_empty() {
+            packages
```

Comment:
nitty nitpick: This could probably be stated as:

```rust
if self.only.is_empty() {
  return packages;
}
... // Contents of "else" block
```

To reduce some indentation